services/application.js

```diff
@@ -191,6 +197,20 @@ class Application {
             url: this.emailUrl
         })
     }
+
+    async sendEmailAfterApproveApplication(context, application) {
+        await this.notificationService.approveQuestionNotification(application?.primaryContact?.email,
+            // Organization Owner and concierge assigned/Super Admin
+            config.temp_email_receiver,
+        {
+            firstName: application?.applicantName
+        }, {
+            study: application?.study?.name,
+            doc_url: config.temp_test_url,
+            org_owner_email: config.temp_email_receiver,
+            concierge_email: config.temp_email_receiver
```

Comment:
Getting org owner and concierge email will require new user operations. I will create a task to implement this that will be need to be completed before we can finish this change.

Reply:
The below logic should be added to determine who should be in the CC list

if the user does not have an org or the org does not have a concierge then CC the admin else CC the concierge
if the user has an org and the org has an owner then CC the org owner
Pass empty strings as the message parameters for org owner or concierge if they are missing